docs/server/source/production-deployment-template/troubleshoot.rst

```diff
@@ -0,0 +1,77 @@
+Cluster Troubleshooting
+=======================
+
+This page describes some basic issues we have faced while deploying and
+operating the cluster.
+
+1. MongoDB Restarts
+-------------------
+
+We define the following in the ``mongo-ss.yaml`` file:
+
+.. code:: bash
+
+    resources:
+      limits:
+        cpu: 200m
+        memory: 5G
+
+When the MongoDB cache occupies a memory greater than 5GB, it is
+terminated by the ``kubelet``.
+This can usually be verified by logging in to the worker node running MongoDB
+container and looking at the syslog (the ``journalctl`` command should usually
+work). It should show an error message such as:
```

Comment:
Such as what? Maybe just remove the "It should show an error message such as:"

At a higher level, this issue never actually happened with the 5GB case, as far as I know. PR #1757 is what bumped it up to 5GB. Maybe say that (i.e. it happened when "memory: 3.5G" but not 5G).